devel/goflymake/Portfile

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/dougm/goflymake 3b9634ef394a5ec125c6847195b1101ec1f47708
+categories          devel
+maintainers         {@graywolf wolfsden.cz:~+macports} openmaintainer
+license             Unknown
```

Comment:
this isn't recognized nor acceptable. Please investigate what the license is or ask upstream for a clarification.